### DISCUSSION TITLE
Topics of Groups A and B

### DISCUSSION THREAD

**Discussion by massteff** [heig-vd-dai-course/heig-vd-dai-course#197] [Q&A]
Hello @ludelafo @hadrylouis

I asked some friends to send me the slides that were covered in the other DAI course. I found some interesting topics that we didn't touch upon at all... I would like to know if these topics are going to be included. Here are some examples that I found:

<img width="780" alt="image" src="https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/42839432/b5a07232-5afb-43b3-b324-981ca2fdb66e">

![image](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/42839432/6542dbe2-045e-4a47-83e5-7878afa82b66)

I will add @heig-vd-dai-course/students to follow this conversation

**Comment by ludelafo** [comment]
Hello @massteff,

Thanks for your message! 

All topics were discussed in detail in advance with the respective teachers.

While building the course, we both always have looked at each other teacher's slides/course material to ensure we covered the same main topics. Some specific topics were explored more in depth by each teacher.

As for my course, I considered Little's Law interesting but more suitable to a cloud computing course. HTTP version features such as multiplexing in HTTP/2 is interesting as well but as mentioned in the course material, the focus was on HTTP/1.1. I didn't want to add more knowledge than what's currently available: the HTTP chapter is already quite dense as it is. I preferred to stay high level and focus on the main aspects of HTTP.

Therfore, these specific (yet interesting) topics will not be in the final exam. All validated topics by both teachers are mentioned in the semester review and exam preparation presentation (https://heig-vd-dai-course.github.io/heig-vd-dai-course/26-semester-review-and-exam-preparation/), except the ones that are explicitly mentioned not to be in the exam.

Does that answer your question?

**Reply by massteff** [reply]
Hello @ludelafo,

All good for me.

Thanks you very much!
